build.sbt

```diff
@@ -22,9 +28,9 @@ description := {
 }
 
 // Developed against 2.10, see crossrelease.sh for all tested and build versions.
```

Comment:
So 2.11 will be the new development version. That's fine by me. Can you please update the comment?